docs/user-interface/graphics/index.md

```diff
@@ -0,0 +1,76 @@
+
+- `Microsoft.Maui.Graphics` functionality must be consumed on a drawing canvas, enables performant graphics to be drawn, and provides a convenient approach for writing graphics-based controls. For example, a control that replicates the GitHub contribution profile can be more easily implemented using `Microsoft.Maui.Graphics` than by using .NET MAUI shapes.
+- .NET MAUI shapes can be consumed directly on a page, and brushes can be consumed by all controls. This functionality is provided to help you produce an attractive UI.
+
+## Platform abstractions
+
+The following table lists the graphics abstractions that are supported on each platform:
+
+| Platform | Graphics abstractions |
+| -- | -- |
+| .NET MAUI | Platform support as shown per platform below. |
+| .NET for iOS | CoreGraphics, SkiaSharp |
+| .NET for Android | Android.Graphics, SkiaSharp |
+| .NET for macOS | CoreGraphics, SkiaSharp |
+| Windows Presentation Foundation | SharpDX, XAML, GDI, SkiaSharp |
+| Universal Windows Platform | SharpDX, Win2D, XAML, SkiaSharp |
+| Windows Forms | SharpDX, GDI, SkiaSharp |
+| Tizen | SkiaSharp |
+| Linux | SkiaSharp |
+
+By default, .NET MAUI uses the native graphics capabilities of each platform.
+
+For examples of using different graphics abstractions, see [Microsoft.Maui.Graphics samples](https://github.com/dotnet/Microsoft.Maui.Graphics/tree/main/samples).
+
+## Drawing canvas
+
+In .NET MAUI, the `GraphicsView` enables consumption of the `Microsoft.Maui.Graphics` functionalit, via a drawing canvas that's exposed as an `ICanvas` object. For more information about the `GraphicsView`, see [GraphicsView](~/user-interface/controls/graphicsview.md).
+
+`ICanvas` defines the following properties, that affect the appearance of object's that are drawn on the canvas:
+
+- `Alpha`, of type `float`, indicates the opacity of an object.
+- `Antialias`, of type `bool`, specifies whether anti-aliasing is enabled.
+- `BlendMode`, of type `BlendMode`, defines the blend mode, which determines what happens when an object is rendered on top of an existing object.
+- `DisplayScale`, of type `float`, represents the scaling factor to scale the UI by on a canvas.
+- `FillColor`, of type `Color`, indicates the color used to paint an object's interior.
+- `FontColor`, of type `Color`, specifies the font color when drawing text.
+- `FontName`, of type `string`, defines the font when drawing text.
+- `FontSize`, of type `float`, defines the size of the font when drawing text.
+- `MiterLimit`, of type `float`, specifies the limit of the miter length of line joins in an object.
+- `StrokeColor`, of type `Color`, indicates the color used to paint an object's outline.
+- `StrokeDashPattern`, of type `float[]`, specifies the pattern of dashes and gaps that are used to outline an object.
+- `StrokeLineCap`, of type `LineCap`, describes the shape at the start and end of a line.
```

Comment:
To align properties and avoid duplicates types between Graphics and Maui, the type of some properties could change.